src/test/spec/oidc.rs

```diff
@@ -157,6 +169,7 @@ async fn machine_1_2_callback_is_called_only_once_for_multiple_connections() ->
     Ok(())
 }
 
+#[cfg(not(feature = "azure-oidc"))]
```

Comment:
Is the idea here to _only_ run the azure tests if `azure-oidc` is enabled?  If so, it seems like it'd be better to put them in their own module that can be targeted by name.

Reply:
yes, and that sounds good to me.